packages/logging-interceptor/src/logging.interceptor.ts

```diff
@@ -209,4 +264,48 @@ export class LoggingInterceptor implements NestInterceptor {
 
     return parsedData;
   }
+
+  /**
+   * Mask the given headers
+   * @param headers the headers to mask
+   * @returns the masked headers
+   */
+  private maskHeaders(headers: IncomingHttpHeaders): Record<string, unknown> {
```

Comment:
```suggestion
  private maskHeaders(headers: IncomingHttpHeaders): Record<string, unknown> {
  
    if (this.disableMasking) {
      return headers;
    }

```

Or even:

```suggestion
  private maskHeaders(headers: IncomingHttpHeaders): Record<string, unknown> {
  
    if (this.disableMasking || this.mask?.requestHeader === undefined) {
      return headers;
    }

```

Reply:
Nice catch! I forgot that case. I will fix it.